### PR TITLE
Removes static membership list

### PR DESCRIPTION
**Pull request by anvega**
For several months we have discussed that while a static membership list is a good way to recognize historic attendance it is not entirely representative of active participation. Given the group dynamics, the group membership is fairly fluid and constantly evolving. Active members recognize each other by their participation and contributions. Doing away with the list is in the spirit of the nature of our group. In terms of the list, there is some redundancy with GitHub contributors. We also understand that not all forms of contribution are directly made in form of code or papers merged to the repository, we'd like to find incentivize and encourage active members to engage on projects and issues as means of publicly sharing and showing their contributions to the broader community.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,8 +129,7 @@ list below:
 
 ## New members
 
-If you are new to the group, check out our [New Members Page](NEW-MEMBERS.md) and
-submit a PR to add yourself to the [members list](#stag-members).
+If you are new to the group, we encourage you to check out our [New Members Page](NEW-MEMBERS.md.
```
Comment (ashutosh-narkar) [inline README.md:132]: Nit: Fix link `[New Members Page](NEW-MEMBERS.md)`

File: README.md [hunk 2]
```diff
@@ -129,8 +129,7 @@ list below:
 
 ## New members
 
-If you are new to the group, check out our [New Members Page](NEW-MEMBERS.md) and
-submit a PR to add yourself to the [members list](#stag-members).
+If you are new to the group, we encourage you to check out our [New Members Page](NEW-MEMBERS.md.)
```
Comment (lumjjb) [inline README.md:132]:
```suggestion
If you are new to the group, we encourage you to check out our [New Members Page](NEW-MEMBERS.md)
```
Link checker ftw

**Review by lumjjb — COMMENTED** [review]
link broken

**Review by lumjjb — APPROVED** [review]
LGTM waiting on CI to pass

**Comment by PushkarJ** [comment]
All checks passed. Merging

**Pull request merged**